pkg/api/v1alpha1/cloudstackmachineconfig_types.go

```diff
@@ -32,6 +32,8 @@ type CloudStackMachineConfigSpec struct {
 	ComputeOffering CloudStackResourceIdentifier `json:"computeOffering"`
 	// Users consists of an array of objects containing the username, as well as a list of their public keys. These users will be authorized to ssh into the machines
 	Users []UserConfiguration `json:"users,omitempty"`
+	// Defaults to `no`. Can be `pro` or `anti`. Will create an affinity group per machine set.
```

Comment:
I think this description could use more detail. "If set, will create an affinity group per machine set of the corresponding type"